bin/postgres-ha/bootstrap/post-bootstrap.sh

```diff
@@ -20,29 +20,33 @@ enable_debugging
 
 echo_info "postgres-ha post-bootstrap starting"
 
-# Run either a custom or the defaul setup.sql file
-if [[ -f "/pgconf/setup.sql" ]]
+if [[ "${PGHA_BOOTSTRAP_METHOD}" == "initdb" ]]
 then
-    echo_info "Using custom setup.sql"
-    cp "/pgconf/setup.sql" "/tmp"
+    # Run either a custom or the defaul setup.sql file
+    if [[ -f "/pgconf/setup.sql" ]]
+    then
+        echo_info "Using custom setup.sql"
+        setup_file="/opt/cpm/bin/sql/setup.sql"
+    else
+        echo_info "Using default setup.sql"
+        setup_file="/opt/cpm/bin/sql/setup.sql"
+    fi
 else
-    echo_info "Using default setup.sql"
-    cp "/opt/cpm/bin/sql/setup.sql" "/tmp"
+    if [[ -f "/pgconf/post-existing-init.sql" ]]
+    then
+        echo_info "Using custom post-existing-init.sql"
+        setup_file="/pgconf/post-existing-init.sql"
+    else
+        echo_info "Using default post-existing-init.sql"
+        setup_file="/opt/cpm/bin/sql/post-existing-init.sql"
+    fi
+    # make sure backrest is stopped before writing to the db in the even we are connected to an
+    # active repostiory for another cluster, such as when bootstrapping using 'pgbackrest_init'
+    pgbackrest stop
```

Comment:
Does this do something to the repo or does it only affect something local? When is the effect reversed?

Reply:
It just writes a stop file to the `tmp` directory that prevents and pgBackRest commands from running.  So when the new container deploys via Deployment to fully initialize the cluster, the lock will be gone.

Reply:
```bash
bash-4.2$ pgbackrest stop
bash-4.2$ ls /tmp/pgbackrest
db.stop
```